### PR TITLE
Custom modal

### PR DESCRIPTION
**Pull request by dspencer001**
Use a custom modal for all uploads, instead of the tinymce one.

### CODE REVIEW

File: client/js/_author/components/common/editor_upload_modal.jsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+const tagNameMap = {
+  audio: 'Audio',
+  img: 'Image',
+  video: 'Video',
+};
+
+export default function EditorUploadModal(props) {
+  function insertMedia() {
+    if (!props.mediaUrl) return;
```
Comment (BenHSmith13) [inline client/js/_author/components/common/editor_upload_modal.jsx]: Do we need this to render, or can we move this logic to a parent component

Comment (dspencer001) [inline client/js/_author/components/common/editor_upload_modal.jsx]: This just says if we don't have a url, don't put  inside of tinymce

File: client/js/_author/components/common/editor_upload_modal.jsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    props.editor.insertContent(editorContent);
+  }
+
+  return (
+    <Modal
+      overlayClassName="author--c-wysiwyg-modal-background"
+      className="author--c-wysiwyg-modal"
+      isOpen={props.isOpen}
+      onRequestClose={props.closeModal}
+      contentLabel="Example Modal"
```
Comment (dspencer001) [inline client/js/_author/components/common/editor_upload_modal.jsx]: This should be "Insert Media Modal" or something like it

File: client/js/_author/components/common/editor_upload_modal.jsx (hunk 1 above)
Comment (dspencer001) [inline client/js/_author/components/common/editor_upload_modal.jsx]: Nevermind, yeah, I get what you were saying now. I moved it up.

**Pull request merged**